blueprints/ember-bootstrap-controls/index.js

```diff
@@ -1,4 +1,13 @@
 module.exports = {
   description: 'Ember Bootstrap Controls Install blueprint',
   normalizeEntityName: function() {},
+  afterInstall() {
+    // Add addons to package.json and run defaultBlueprint
+    return this.addAddonsToProject({
+      // a packages array defines the addons to install
+      packages: [
+        {name: 'ember-cli-bootstrap-4'},
```

Comment:
We've been adding bootstrap using https://www.ember-bootstrap.com/, does this library conflict with that one? If so, how do we want to recommend updating our internal applications?